core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -647,4 +647,42 @@ package object config {
       .stringConf
       .toSequence
       .createWithDefault(Nil)
+
+  private[spark] val KAFKA_DELEGATION_TOKEN_ENABLED =
```

Comment:
This seems a little redundant. If you set `KAFKA_BOOTSTRAP_SERVERS` - which is optional - and the protocol needs tokens, then this can be assumed to be true.

Reply:
Removed.